README.md

```diff
@@ -86,6 +86,15 @@ accessible URL for these results. For example:
   "results": {
     "destination": "user@example.org:public_html/logs",
     "public_url": "https://example.com/logs"
+  },
+  "logging": {
+    "level": "WARNING",
+    "filename": "/var/log/test-harness_HOSTNAME.log",
```

Comment:
Note that when running in a container, this file will be inside the container, so you will have to do something like `oc exec $pod-name -- cat /var/log/test-harness_HOSTNAME.log` in order to view the contents (or `oc rsh $pod-name` to login to the pod and manually navigate).